packages/contracts/test/dollar/Staking.t.sol

```diff
@@ -166,14 +167,16 @@ contract RemoteZeroStateTest is ZeroState {
         vm.stopPrank();
 
         assertEq(
-            bytes20(address(steak)), bytes20(address(staking.stakingFormulas()))
+            bytes20(address(steak)),
```

Comment:
I'm pretty sure I fixed this in your previous PR, you might need to rebase to a later commit. Its stake, not steak.

Reply:
It appears this was resolved when I rebased from your PR that fixed CI